packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +75,63 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+// Custom hook to evaluate an array of expressions using useEvaluate
+const useEvaluateExpressions = (
+  template: CollapsibleItem,
+  namedData: object[],
+): CollapsibleTemplate[] => {
+  const [evaluatedItems, setEvaluatedItems] = useState<CollapsibleTemplate[]>(
+    [],
+  );
+  const [currentIndex, setCurrentIndex] = useState(-1);
```

Comment:
Do not do this! Why did you no longer make this a component? This is going to be `n` renders of the entire Collapsible component instead of individual smaller renders.

Change this back to a component. Add a check in the parent Collapsible component so that it doesn't render anything until all the keys have been evaluated.